rolling/release-rhel-build.yaml

```
@@ -26,6 +26,7 @@ package_blacklist:
 - gazebo_dev  # Gazebo has no RPM package
 - grbl_ros  # Not yet generated for RHEL
 - ign_rviz_common  # ignition has no RPM packages for RHEL
+- joint_state_broadcaster # Build failures on RHEL
```

nit: to pass yamllint

```suggestion
- joint_state_broadcaster  # Build failures on RHEL
```